eng/pipelines/templates/steps/set-dev-build.yml

```diff
@@ -3,7 +3,7 @@ parameters:
   ServiceDirectory: ''
 
 steps:
-  - template: tools/daily-dev-build-variable/daily-dev-build-variable.yml@azure-sdk-tools
+  - template: /eng/common/pipelines/templates/steps/daily-dev-build-variable.yml
```

Comment:
Is this present on `release/v3`?

Reply:
Please ignore this change. I created this draft PR to investigate analyze failure. Pylint failure is not reproducing locally so needed to get pip freeze. But it seems my previous changes in local also got pushed.